CMakeLists.txt

```diff
@@ -5,10 +5,10 @@ project(inexor-vulkan-renderer CXX)
 set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
 
 # Options
-option(INEXOR_BUILD_BENCHMARKS "Build benchmarks" OFF)
+option(INEXOR_BUILD_BENCHMARKS "Build benchmarks" ON)
 option(INEXOR_BUILD_DOC "Build documentation" OFF)
 option(INEXOR_BUILD_EXAMPLE "Build example" ON)
-option(INEXOR_BUILD_TESTS "Build tests" OFF)
+option(INEXOR_BUILD_TESTS "Build tests" ON)
```

Comment:
I would keep them all at off by default, as the regular user might only need the library aka. the engine. If someone or we require more i would enable them via arguments.

Reply:
Hmm okay.